mms/export_model.py

```diff
@@ -294,24 +294,26 @@ def export_model(model_name, model_path, service_file=None, export_file=None):
     signature_file = validate_signature(model_path)
     service_file = validate_service(model_path, service_file, signature_file)
     if os.path.basename(service_file) not in files:
-        files.append(os.path.basename(service_file))
+        temp_files.append(os.path.basename(service_file))
         shutil.copyfile(service_file, os.path.join(model_path, os.path.basename(service_file)))
     service_file = os.path.basename(service_file)
 
     if onnx_file:
         symbol_file, params_file = convert_onnx_model(model_path, onnx_file)
         files.remove(onnx_file)
-        files.extend([symbol_file, params_file])
+        temp_files.extend([symbol_file, params_file])
 
     manifest = generate_manifest(symbol_file, params_file, service_file, signature_file, model_name)
     with open(os.path.join(model_path, MANIFEST_FILE_NAME), 'w') as m:
         json.dump(manifest, m, indent=4)
-    files.append(MANIFEST_FILE_NAME)
+    temp_files.append(MANIFEST_FILE_NAME)
 
     with zipfile.ZipFile(export_file, 'w') as z:
         for f in files:
             z.write(os.path.join(model_path, f), f)
```

Comment:
Can we try to iterate over a combined list of files and temp_files, instead of duplicating the zip write statement?
We can then have a separate loop for removing the temp files.